spk/itools/Makefile

```diff
@@ -0,0 +1,28 @@
+SPK_NAME = itools
+SPK_VERS = 1.0
+SPK_REV = 1
+SPK_ICON = src/itools.png
+
+DEPENDS = cross/usbmuxd cross/ifuse
+UNSUPPORTED_ARCHS = $(ARM_ARCHES) $(PPC_ARCHES)
```

Comment:
What are reasons why ARM and PPC may not run these command line tools?

Reply:
It cannot pass "make all-supported" tests. 
I simply marked them as unsupported models.
Someone can solve it in the future.